components/navigation/BlogPostItem.js

```diff
@@ -25,7 +25,7 @@ export default function BlogPostItem({ post, className = '' }) {
 
   return (
     <Link href={post.slug} passHref>
-      <a className={`flex flex-col border border-gray-200 rounded-lg shadow-md divide-y divide-gray-200 transition-all duration-300 ease-in-out hover:shadow-lg overflow-hidden cursor-pointer ${className}`}>
+      <a className={`flex flex-col border border-gray-200 rounded-lg shadow-md divide-y divide-gray-200 transition-all duration-300 ease-in-out hover:shadow-lg overflow-hidden cursor-pointer ${className}`} style={{minWidth:"22rem"}}>
```

Comment:
For me inline `style` can be applied, of course only in places where tailwind does not have the given class and we use the given styles only once.